contracts/VetoGuard.sol

```diff
@@ -150,6 +134,8 @@ contract VetoGuard is
         bytes memory,
         address
     ) external view override {
+        if (executionDelayBlocks == 0 && address(vetoERC20Voting) == address(0))
+            return;
```

Comment:
I remember this is checking if the contract has been initialized. Can you explain again why this is necessary? Couldn't we just call `setGuard` as the very last function call?

Reply:
Good catch here - since we are calling multisend - we do not have to worry about the possible issue of hitting check transaction.